doc/manual/citing_aspect.bib

```diff
@@ -1411,9 +1411,34 @@ @article{SRIVASTAVA2022105464
 pages = {105464},
 year = {2022},
 issn = {1367-9120},
-doi = {/10.1016/j.jseaes.2022.105464},
+doi = {10.1016/j.jseaes.2022.105464},
 url = {https://www.sciencedirect.com/science/article/pii/S1367912022003959},
 author = {Rajesh K. Srivastava and Fei Wang and Wenbei Shi and Richard E. Ernst},
 keywords = {Early Cretaceous, dolerite dykes, Ar-Ar date, Greater Kerguelen LIP, Eastern India},
 abstract = {Early Cretaceous NNW- to WNW-trending dolerite dykes of the eastern Indian Shield, collectively termed the Raniganj-Koderma swarm, are studied for their emplacement ages and petrogenetic history, and assessed for a possible linkage with the Greater Kerguelen plume. 40Ar/39Ar dates of four dolerite dykes from different locations in the Chhota Nagpur Gneissic Terrane, indicate three pulses of emplacement ca. 118-116 Ma, ca. 112-111 Ma, and ca. 109 Ma. Geochemistry of 19 samples analysed herein (and an additional 12 samples from literature) shows sub-alkaline high-Mg tholeiitic basaltic andesite compositions. All the dyke sets belonging to the Raniganj-Koderma swarm with sub-sets WNW-trending ca. 118 Ma, NNW-trending ca. 116 Ma Salma, and a N-S trending ca. 109 Ma dyke show similar chemistry. However, NW-trending dykes emplaced ca. 112-111 Ma have slightly different geochemical characteristics. Key trace element geochemistry data, particularly Nb/Y, Zr/Y, Nb/Yb, Ti/Yb, Th/Nb and Th/Yb ratios, indicates derivation from mantle melts generated from interaction of a plume with a spreading ridge (producing OIB – E-MORB melts) with a minor role for interaction with an enriched lithospheric mantle metasomatized during an earlier (pre-Mesozoic time) subduction event. Spatiotemporal distribution of the studied dolerite dykes connects them with the second plume head phase of the Greater Kerguelen mantle plume.}
+}@article{https://doi.org/10.1029/2022GL101130,
+author = {Dong, Miao and Lü, ChuanChuan and Zhang, Jian and Hao, Tianyao},
```

Comment:
Do we need to escape the umlaut in "Lü"?

Reply:
Funny things do happen.  Thanks.